openshift2nulecule/cli/main.py

```diff
@@ -46,19 +46,52 @@ def __init__(self):
                                  help="Show debug messages",
                                  action='store_true')
 
+        self.parser.add_argument("--oc-registry-host",
+                                 help="Hostname of exposed internal OpenShift registry",
+                                 required=False)
+
+        self.parser.add_argument("--export-images",
+                                 help="Pull images from OpenShift registry to "
+                                      "local Docker instance and push them to "
+                                      "remote registry (--registry-host) ",
+                                 choices=["none", "internal", "all"],
+                                 default="none",
+                                 required=False)
+
+        self.parser.add_argument("--registry-host",
+                                 help="External registry hostname",
```

Comment:
If `--registry-host` is not specified than images are only pulled by Docker, but won't be pushed to any registry. 

Right now this is always optional even with `--export-images`. Now when I'm thinking about this, maybe this should be required when `--export-images`!=none. 
I can't think of use case when someone would only want to pull images and not to push them somewhere.